.cmake-format.py

```diff
@@ -0,0 +1,230 @@
+# ----------------------------------
+# Options affecting listfile parsing
+# ----------------------------------
+with section("parse"):
+
+  # Specify structure for custom cmake functions
+  additional_commands = {
+    "DLAF_addTest": {
+      "pargs": 1,
+      "flags": [],
+      "kwargs": {
+        "SOURCES": '*',
+        "COMPILE_DEFINITIONS": '*',
+        "INCLUDE_DIRS": '*',
+        "LIBRARIES": '*',
+        "ARGUMENTS": '*',
+        "USE_MAIN": '1',
+        "MPIRANKS": '1',
+      }
+    },
+    "DLAF_addMiniapp": {
+      "pargs": 1,
+      "flags": [],
+      "kwargs": {
+        "SOURCES": '*',
+        "COMPILE_DEFINITIONS": '*',
+        "INCLUDE_DIRS": '*',
+        "LIBRARIES": '*',
+        "ARGUMENTS": '*',
+        "USE_MAIN": '1',
+        "MPIRANKS": '1',
+      }
+    },
+    "DLAF_addPrecompiledHeaders": {
+      "pargs": 1,
+    },
+    "target_add_warnings": {
+      "pargs": 1,
+    }
+  }
+
+  # Override configurations per-command where available
+  override_spec = {}
+
+  # Specify variable tags.
+  vartags = []
+
+  # Specify property tags.
+  proptags = []
+
+# -----------------------------
+# Options affecting formatting.
+# -----------------------------
+with section("format"):
+
+  # Disable formatting entirely, making cmake-format a no-op
+  disable = False
+
+  # How wide to allow formatted cmake files
+  line_width = 105
+
+  # How many spaces to tab for indent
+  tab_size = 2
+
+  # If true, lines are indented using tab characters (utf-8 0x09) instead of
+  # <tab_size> space characters (utf-8 0x20). In cases where the layout would
+  # require a fractional tab character, the behavior of the  fractional
+  # indentation is governed by <fractional_tab_policy>
+  use_tabchars = False
+
+  # If an argument group contains more than this many sub-groups (parg or kwarg
+  # groups) then force it to a vertical layout.
+  max_subgroups_hwrap = 2
```

Comment:
```suggestion
  max_subgroups_hwrap = 3 (or 4)
```
Maybe? This makes some more function calls fit on one line. Not a must as this is just my preference though...

Reply:
I prefer 3 over 2. With 4 that output is a bit surprising and ugly.